src/main/java/org/app/controller/TodoController.java

```diff
@@ -29,7 +30,7 @@ public ResponseEntity<TaskResponseDto> getTask(@PathVariable("id") Long id, Auth
         return ResponseEntity.ok(task);
     }
 
-    @PutMapping("/{id}")
+    @PutMapping("/{id}") // todo
```

Comment:
Suppose I want to change only the task's "completed" field. I'd get an error in this case